bcrs-ui/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>hello-world</title>
+    <link rel="icon" href="dummy-not-used-do-not-remove">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="preload" href="<%= BASE_URL %>fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="stylesheet" href="<%= BASE_URL %>css/addresscomplete-2.00.min.css">
+    <script type="text/javascript" src="<%= BASE_URL %>js/addresscomplete-2.00.min.js" defer></script>
```

Comment:
Please check in these 2 files above. (I think we'll probably want to use AC.)

Reply:
Sounds good to me, out of curiosity was AddressComplete part of the app spec or was it something we chose. We're there other options out there? ( NOT suggesting we change, just curious! )

Reply:
Dunno. In fact we're not using it to its potential (where it provides a key indicating the entered address is valid). I'll let the business drive any changes here...

Reply:
Still need to either remove these 2 AddressComplete lines or check in the 2 files.

Reply:
Will remove until we need 👍